example/lib/main.dart

```diff
@@ -35,7 +35,7 @@ class MyApp extends StatelessWidget {
                     context: context,
                     builder: (_) {
                       return MailAppPickerDialog(
-                        mailApps: result.options,
+                        mailApps: result.options!,
```

Comment:
Should we make the options list returned non nullable? It would throw at this point if it was.

The list can be empty, but if we make it non nullable it makes it a lot safer to pass into this dialog and for other uses users may want.

If we detect it as null, we can assign empty list.

Reply:
done :)